crates/gitbutler-tauri/src/env.rs

```diff
@@ -0,0 +1,7 @@
+use std::collections::BTreeMap;
+
+#[cfg(debug_assertions)]
```

Comment:
Probably worth adding a comment that this should not be removed (i.e we definitely do not want this in prod)